framework/src/araui/core/ui_core.c

```diff
@@ -739,12 +712,18 @@ ui_error_t ui_core_quick_panel_appear(ui_quick_panel_event_type_t event_type)
 		break;
 	}
 
+	if (move) {
+		ui_anim_destroy(move);
```

Comment:
By this line,
The instance of the move animation will be deleted right after created.
This may causes memory leak or access violation.

Reply:
`ui_anim_destroy` is moved inside of each `anim_finished_cb`